pkg/plumber/types.go

```diff
@@ -74,9 +45,6 @@ type PlumberJobSpec struct {
 	// ExtraRefs are auxiliary repositories that
 	// need to be cloned, determined from config
 	ExtraRefs []Refs `json:"extra_refs,omitempty"`
```

Comment:
Where are we using this `ExtraRefs` argument?

Reply:
removed